connect-examples/oauth/php/README.md

```diff
@@ -1,15 +1,15 @@
-# Connect OAuth Flow Example (PHP)
+# Square OAuth Flow Example (PHP)
 
 This example demonstrates a bare-bones PHP implementation of the OAuth flow for
 Square APIs. It serves a link that directs merchants to the OAuth Permissions form
 and handles the result of the authorization, which is sent to your application's
 Redirect URL (specified on the application dashboard).
 
-For more information, see [OAuth Overview](https://docs.connect.squareup.com/api/oauth#oauth-overview), along with the comments included in `callback.php`.
+For more information, see [OAuth Overview](https://docs.connect.squareup.com/api/oauth#oauth-overview), along with the comments included in `sandbox_callback.php`.
```

Comment:
Link will be wrong. Get the link to the walkthrough and add it here. Have John get that. Text should be: For a detailed walkthrough of this example, see OAuth Walkthrough.

Reply:
updated.